modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/groupby/BaseGroupByTestCase.java

```diff
@@ -352,15 +348,13 @@ public void testGroupByTermsSizeMoreThanDefault() throws Exception {
 
 		map1.forEach((key, value) -> indexDuplicates(key, value));
 
-		Set<Map.Entry<String, Integer>> entries = map1.entrySet();
-
-		Stream<Map.Entry<String, Integer>> stream = entries.stream();
+		Map<String, String> map2 = new HashMap<>();
 
-		Map<String, String> map2 = stream.collect(
-			Collectors.toMap(
-				Map.Entry::getKey,
-				entry -> getCountPairString(
-					entry.getValue(), entry.getValue())));
+		for (Map.Entry<String, Integer> entry : map1.entrySet()) {
+			map2.computeIfAbsent(
```

Comment:
@julschong No need to computeIfAbsent here.

Your source is the entry set of another map, the keys won't have duplicate.